src/app/(tabs)/index.tsx

```diff
@@ -43,12 +50,25 @@ const HomePage = () => {
   if (!isSetupComplete) {
     return <Redirect href="/setup/start" />;
   }
+
   const toggleBackgroundFetch = () => {
     if (setTrackingEnabled) {
       setTrackingEnabled(!isTrackingEnabled);
     }
   };
 
+  const handleSendNotification = async () => {
+    try {
+      await sendPushNotification({
```

Comment:
this try catch is not needed here, because it is already handled inside the sendPushNotification